.vscode/settings.json

```diff
@@ -10,6 +10,9 @@
   "files.trimFinalNewlines": true,
   "files.insertFinalNewline": true,
   "editor.detectIndentation": false,
-  "editor.tabSize": 2,
-  "editor.insertSpaces": true
+  "editor.tabSize": 4,
+  "editor.insertSpaces": true,
+  "explorer.fileNesting.patterns": {
+    "*.cs": "I${capture}.cs",
```

Comment:
Do you mind explaining this a bit? Does this actually alter directory structure or is it purely cosmetic if you have the extension?

Reply:
Purely cosmetic.  It only affects the built-in vscode [**Explorer**](https://code.visualstudio.com/docs/getstarted/userinterface#_explorer-view) **AND** _only_ if the user already has [`explorer.fileNesting.enabled: true`](https://code.visualstudio.com/updates/v1_67#_explorer-file-nesting) in their user settings.  If they do, it renders with the interface with a matching name grouped together with the class.

![Screenshot 2024-06-28 at 11 44 27 PM](https://github.com/octokit/octokit.net/assets/1829082/9438670b-3f80-49ce-99de-28a325f6cf07)

It has no effect on the [**Solution Explorer**](https://code.visualstudio.com/docs/csharp/project-management#_solution-explorer) (that comes with the C# Dev Kit extension)

![Screenshot 2024-06-28 at 11 42 56 PM](https://github.com/octokit/octokit.net/assets/1829082/4b10cca6-e7d9-4d95-89bb-4a0f2612fa96)